core/particles/main.zig

```diff
@@ -0,0 +1,245 @@
+const std = @import("std");
```

Comment:
Did you mean to send this in this PR? cool either way

Reply:
that isn't ready yet. lol